aya/src/util.rs

```diff
@@ -95,6 +95,34 @@ fn parse_kernel_symbols(reader: impl BufRead) -> Result<BTreeMap<u64, String>, i
     Ok(syms)
 }
 
+/// Returns the prefix used by syscalls.
+/// ## Example
+/// ```no_run
+/// let prefix = syscall_prefix()?;
+/// let syscall_fname = format!("{prefix}exec");
+/// ```
+/// ## Errors
```

Comment:
Newline before and after please, also # not ##

Also the doc is wrong, it returns other error kinds as well. It should say:

```
Returns [`std::io::ErrorKind::NotFound`] if the prefix can't be guessed. Returns other [`std::io::Error`] kinds if `/proc/kallsyms` can't be opened or is somehow invalid.
```